sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
It shouldn't matter since they all bring the same dependency, and I believe that `WritableCoder` should be in `hadoop-common` and not `hdfs-io`,